concourse/pipelines/deploy.yml

```diff
@@ -196,6 +182,25 @@ resources:
       versioned_file: router.json
       initial_version: "0"
 
+
+  - name: frontend-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: govuk/frontend
+      tag: deployed-to-production
+      username: ((docker_hub_username))
+      password: ((docker_hub_authtoken))
+
+  - name: static-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: docker.io/govuk/static
```

Comment:
Why has this one got `docker.io` as a prefix, when frontend doesn't?

Reply:
I can fix that.